Documentation/getting-started-test-app.md

```diff
@@ -1,5 +1,31 @@
 # Getting started with the test app
 
+**Data Collection mobile property prerequisites**
+
+The test app needs to be configured with the following edge extensions before it can be used:
+- Mobile Core (installed by default)
+- [Edge](https://github.com/adobe/aepsdk-edge-android)
+- [Edge Identity](https://github.com/adobe/aepsdk-edgeidentity-android)
+- [Edge Consent](https://github.com/adobe/aepsdk-edgeconsent-android) (recommended when using the setAdvertisingIdentifier API)
+
+1. In the test app, set your `ENVIRONMENT_FILE_ID` in `EdgeIdentityApplication.kt`.
+2. Select the `app` runnable with the desired emulator and run the program.
+
+> **Note**
+> To enable GAID related advertising identifier features, follow the [documentation](./advertising-identifier.md) for the required setup steps.
+
+**View the platform events with Assurance**
+
+Configure a new Assurance session by setting the Base URL to `testapp://main` and launch Assurance in the demo app by running the following command in your terminal:
+
+```bash
+$ adb shell am start -W -a  android.intent.action.VIEW -d "testapp://main?adb_validation_sessionid=ADD_YOUR_SESSION_ID_HERE" com.adobe.marketing.mobile.testapp
```

Comment:
looks like this test app does not have the scheme configured for deeplinks. e.g.:
https://github.com/adobe/aepsdk-edge-android/blob/dev-v2.0.0/code/app/src/main/AndroidManifest.xml#L40-L43

https://github.com/adobe/aepsdk-edge-android/blob/dev-v2.0.0/code/app/src/main/java/com/adobe/marketing/tester/MainActivity.java#L121-L128